topo/node/cptx/cptx_test.go

```diff
@@ -139,6 +140,44 @@ func TestConfigPush(t *testing.T) {
 	}
 }
 
+func TestCustomPrivilegeLevel(t *testing.T) {
+	tests := []struct {
+		desc        string
+		hostname    string
+		shouldMatch bool
+	}{
+		{
+			desc:        "basic case",
+			hostname:    "testexample",
+			shouldMatch: true,
+		},
+		{
+			desc:        "hostname with '.'",
+			hostname:    "test.example",
+			shouldMatch: true,
+		},
+	}
+
+	privilegePromptMap := map[string]string{
```

Comment:
nit: usually the pattern for tests that we follow is:

things needed for all test cases

testcase def

run tests

So in this case
```
privilegePromptMap := ...

tests := []struct...
for _, tt := range tests ...
```